bpf/kmesh/probes/metrics.h

```diff
@@ -114,7 +128,6 @@ metric_on_close(struct bpf_sock *sk, struct bpf_tcp_sock *tcp_sock, struct sock_
     metric = (struct metric_data *)bpf_map_lookup_elem(&map_of_metrics, &key);
     if (!metric) {
         // connect failed
-        data.direction = storage->direction;
         data.conn_failed++;
         int err = bpf_map_update_elem(&map_of_metrics, &key, &data, BPF_NOEXIST);
```

Comment:
Why not BPF_ANY here? BPF_NOEXIST means create a element if i=t didnot exist, obviously it exist in L114